source/MaterialXRuntime/Private/PvtStageManager.cpp

```diff
@@ -0,0 +1,54 @@
+//
+// TM & (c) 2020 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#include <MaterialXRuntime/Private/PvtStageManager.h>
+
+#include <MaterialXRuntime/RtStage.h>
+
+namespace MaterialX
+{
+
+PvtStageManager::PvtStageManager()
+{
+}
+
+RtObject PvtStageManager::createStage(const string& stageName)
```

Comment:
Creates a stage, for now the name must be unique, in the future we should increment the name if the suggested name exists.